packages/shared_preferences/shared_preferences/lib/shared_preferences.dart

```diff
@@ -128,6 +128,15 @@ class SharedPreferences {
       _setValue('Double', key, value);
 
   /// Saves a string [value] to persistent storage in the background.
+  ///
+  /// Note: Due to limitations in Android's SharedPreferences,
+  /// values cannot start with either one of the following:
+  ///
+  /// 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBhIGxpc3Qu'
+  ///
+  /// 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBCaWdJbnRlZ2Vy'
+  ///
+  /// 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBEb3VibGUu'
```

Comment:
Nit: Having all these blank lines seems confusing, since they are all a single list. How about:

```
/// [...] any of the following:
/// - 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBhIGxpc3Qu'
/// - 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBCaWdJbnRlZ2Vy'
/// - 'VGhpcyBpcyB0aGUgcHJlZml4IGZvciBEb3VibGUu'

Reply:
Done in the new commit.